examples/sample-app/application-template-pullspecbuild.json

```diff
@@ -369,6 +377,12 @@
                     "protocol": "TCP"
                   }
                 ],
+                "volumeMounts": [
+                  {
+                    "name": "data",
+                    "mountPath": "/var/lib/mysql/data"
+                  }
+                ],
```

Comment:
looks like the rebase went awry?  there are two volumeMounts stanzas in this container (same for some of the other templates)